port: /dev/ttyUSB0
read_rate: 15
write_rate: 25
joints: {
    head_pan_joint: {id: 19, invert: false, neutral: 512, max_speed: 80},
    head_tilt_joint: {id: 20, max_angle: 150, min_angle: -150, max_speed: 80},
    right_sho_pitch: {id: 1, max_angle: 150, min_angle: -150, max_speed: 80},
    right_sho_roll: {id: 3, max_angle: 150, min_angle: -150, max_speed: 80},
    right_elbow: {id: 5, max_angle: 150, min_angle: -150, max_speed: 80},
    left_sho_pitch: {id: 2, max_angle: 150, min_angle: -150, max_speed: 80},
    left_sho_roll: {id: 4, max_angle: 150, min_angle: -150, max_speed: 80},
    left_elbow: {id: 6, max_angle: 150, min_angle: -150, max_speed: 80}
}
controllers: {
    head_controller: {
        type: follow_controller, 
        joints: [head_pan_joint ,head_tilt_joint,
        right_sho_pitch, right_sho_roll, right_elbow,
        left_sho_pitch, left_sho_roll, left_elbow],
        action_name: head_controller/follow_joint_trajectory,
    }
}

